test/Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -1183,9 +1183,8 @@ public void AbortingTheConnectionSendsFIN()
             }
         }
 
-        [Theory]
-        [MemberData(nameof(ConnectionAdapterData))]
-        public async Task HeadersAndStreamsAreReusedAcrossRequests(ListenOptions listenOptions)
```

Comment:
This one is interesting. We've been running this test twice, but listenOptions was never used to make the test inputs different. I traced this one back to a massive refactoring in our tests added by @halter73 here: https://github.com/aspnet/KestrelHttpServer/pull/1280